Add missing argument and improve documentation (#830)
citizenfx/natives: NETWORK/NetworkFadeInEntity.md

```c
// 0x1F4ED342ACEFE62D 0x9B9FCD02

> void NETWORK_FADE_IN_ENTITY(Entity entity, BOOL flash, BOOL slow);

Reviewer: At this time adding new arguments in this fashion in not supported. Maybe make a note of it in the native description? Thanks!

```

Fade the given entity back in, usually used after the entity has been faded out with [NETWORK_FADE_OUT_ENTITY](#_0xDE564951F95E09ED)

When used on a entity which isn't invisible or faded out then the native will still work, it will just instanly make the ped invisible before fading in.

## Parameters
* **entity**: The entity to fade in
* **flash**: When set to true the entity will flash rapidly while fading out

> * **slow**: When set to true the fadein will be a little bit slower than normal

Reviewer: Remove once you remove the native function declaration argument mentioned up above or it won't compile.

```c
// 0x1F4ED342ACEFE62D 0x9B9FCD02

> void NETWORK_FADE_IN_ENTITY(Entity entity, BOOL flash);

Reviewer 2: ```suggestion
void NETWORK_FADE_IN_ENTITY(Entity entity, BOOL bNetwork);
```

```

Fade the given entity back in, usually used after the entity has been faded out with [NETWORK_FADE_OUT_ENTITY](#_0xDE564951F95E09ED)

When used on a entity which isn't invisible or faded out then the native will still work, it will just instanly make the ped invisible before fading in.

> Note that there is another boolean parameter which is missing, when set to true the fade in will be a little bit slower.

Reviewer: I actually did a full research on this on Sunday, the last parameter is basically a `bSendOverNet` boolean param which basically specifies if the fade should be sent over the network or not.

> Note that there is another boolean parameter which is missing, when set to true the fade in will be a little bit slower.

Reviewer 2: ```suggestion
**Additional Parameters**:
* **flash**: If set to true the entity will flash while fading in.
```

## Parameters
* **entity**: The entity to fade in

> * **flash**: When set to true the entity will flash rapidly while fading out

Reviewer 2: ```suggestion
* **network**: When set to true the entity will flash rapidly while fading out
```

## Examples

```lua

> NetworkFadeOutEntity(PlayerPedId(), false, false)

Reviewer: `NetworkFadeOutEntity` has the boolean parameters reversed, first one is `bSendOverNet` and second one is `bFlash` or `flash`, I just append `b` behind it to specify it's a boolean.

I highlighted the diffs in red.
![diffs](https://github.com/citizenfx/natives/assets/38232208/3104ebf0-47d0-46a5-ae76-961435da5c7e)

What I also noticed is that fading an entity with occupants inside, such as a vehicle, will also fade the occupants, but only if they're classified as net objects (which is basically a net object method that sets the alpha for every entity within the seat manager instance of the specific vehicle entity)

![image](https://github.com/citizenfx/natives/assets/38232208/b3bbad6d-2209-4544-80b9-2c5cfcd29c34)

I improved your current example in order to demonstrate it:
```lua
RegisterCommand("fademe", function(source, args, rawCommand)
    CreateThread(function()
        local entity = GetVehiclePedIsIn(PlayerPedId())
        if entity == 0 then
            entity = PlayerPedId()
        else
            if IsVehicleSeatFree(entity, 0) then
                local playerDummy = ClonePed(PlayerPedId(), true, false, true)
                SetPedIntoVehicle(playerDummy, entity, 0)
            end
        end
       -- Last param is send over net, second is flash
        NetworkFadeOutEntity(entity, false, true)
        while NetworkIsEntityFading(entity) do
            Citizen.Wait(0)
        end
        -- Second parameter here is bSendOverNet, last one is flash
        NetworkFadeInEntity(entity, true, false)
        while NetworkIsEntityFading(entity) do
            Citizen.Wait(0)
        end
    end)
end, false)
```

Note: screenshots are provided for illustration/educational purposes only.

> NetworkFadeOutEntity(PlayerPedId(), false, false)
> while NetworkIsEntityFading(PlayerPedId()) do
>   Citizen.Wait(0)
> end
> 
> --- Do something like a teleport, or warp into a vehicle
> 
> -- Last parameter here is slow, this parameter is missing in decleration
> NetworkFadeInEntity(PlayerPedId(), false, false)
> while NetworkIsEntityFading(PlayerPedId()) do
>   Citizen.Wait(0)
> end
> ```

Reviewer 2: ```suggestion
local playerPed = PlayerPedId()

-- fade out the player while flashing them
NetworkFadeOutEntity(playerPed, true, false)
while NetworkIsEntityFading(playerPed) do
  Wait(0)
end

--- Do something like a teleport, or warp into a vehicle

-- while generally frowned upon when you can use natives, this declaration has
-- a missing parameter, which doesn't work without manually invoking.
Citizen.InvokeNative(0x1F4ED342ACEFE62D, playerPed, false, true)
while NetworkIsEntityFading(playerPed) do
  Wait(0)
end
```

```c
// 0x1F4ED342ACEFE62D 0x9B9FCD02
void NETWORK_FADE_IN_ENTITY(Entity entity, BOOL bNetwork);
```

Fade the given entity back in, usually used after the entity has been faded out with [NETWORK_FADE_OUT_ENTITY](#_0xDE564951F95E09ED)

When used on a entity which isn't invisible or faded out then the native will still work, it will just instanly make the ped invisible before fading in.

**Additional Parameters**:
* **flash**: If set to true the entity will flash while fading in.

## Parameters
* **entity**: The entity to fade in

> * **network**: When set to true the fade in will be networked.

Reviewer 2: ```suggestion
* **bNetwork**: When set to true the fade in will be networked.
```
